test/components/Profile/ProfileImageField.test.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { ProfileImageField } from '../../../src/components/Profile';
+import '@testing-library/jest-dom/extend-expect';
+
+const MockProfileComponent = ({ noUserImage, mockClientProfile }) => {
+  if (!noUserImage) {
+    localStorage.setItem('profileImage', 'https://example.com/user.png');
+  }
+  return <ProfileImageField clientProfile={mockClientProfile} />;
+};
+
+describe('ProfileImageField', () => {
+  it('renders Choose Img button and PersonIcon if clientProfile is null and user has no profile img', () => {
+    const { queryByRole, queryByTestId } = render(
+      <MockProfileComponent noUserImage mockClientProfile={null} />
+    );
+    const buttonElement = queryByRole('button');
+    expect(buttonElement.textContent).toBe('Choose Img');
+
+    const svgElement = queryByTestId('PersonIcon');
+    expect(svgElement).not.toBeNull();
+  });
+
+  it('renders Remove Img button and image if clientProfile is null, but user has profile image', () => {
+    const { queryByRole, queryByTestId } = render(
+      <MockProfileComponent mockClientProfile={null} />
+    );
+    const buttonElement = queryByRole('button');
+    expect(buttonElement.textContent).toBe('Remove Img');
+
```

Comment:
should add an expect for muiAvatar like lines 54 and 55 in this test?

Reply:
Included line for muiAvatar as part of that test:

```
const muiAvatar = queryByRole('img');
expect(muiAvatar).toHaveAttribute("src", "https://example.com/user.png");
```